receiver/jmxreceiver/receiver.go

```diff
@@ -235,7 +235,8 @@ func (jmx *jmxMetricReceiver) buildJMXMetricGathererConfig() (string, error) {
 		config["otel.resource.attributes"] = strings.Join(attributes, ",")
 	}
 
-	var content []string
+	content := make([]string, len(config))
```

Comment:
Let's not track `i` additionally, just change this line
```suggestion
	content := make([]string, 0, len(config))
```